_config.py

```diff
@@ -74,8 +74,8 @@
 
 COOKIE_AGE = 2147483647
 
-# set to true to enable api support
-API_ENABLED = False
+# set to True to enable api support
+API_ENABLED = True

```

Comment:
I think Extravi prefers this being set to `False` by default.

Reply:
i do

Reply:
having an exposed api makes it less reliable